mlir/test/Dialect/GPU/spirv-attach-targets.mlir

```diff
@@ -0,0 +1,7 @@
+// RUN: mlir-opt %s --spirv-attach-target='module=spirv.* ver=v1.0 caps=Kernel' | FileCheck %s
+
+module attributes {gpu.container_module} {
+// CHECK: @spirv_module_1 [#spirv.target<vce = #spirv.vce<v1.0, [Kernel], []>, resource_limits = <>>]
+gpu.module @spirv_module_1 {
```

Comment:
Can we add tests to show like

1) multiple GPU modules in the same container module?
2) different match regex with hits and misses?

Reply:
Done.